src/cli/helpers.js

```diff
@@ -13,6 +13,8 @@ import buildDocumentationObject from '../documentation/build-documentation-objec
 import generateTable from '../documentation/generate-table';
 import { getDefaultValue } from '../documentation/helpers';
 import { fileExists } from '../helpers';
+import { throwError } from '../validation';
+import { isValid } from '../validation/helpers';
```

Comment:
cli helper is importing from yet another helper. Perhaps double check if that function should really reside in a validation-specific helper and not be made more generic.